client/verta/verta/operations/monitoring/profiler.py

```diff
@@ -61,14 +60,14 @@ class ContinuousHistogramProfiler(Profiler):
     def __init__(self, columns, bins=10):
         super(ContinuousHistogramProfiler, self).__init__(columns)
         self._bins = bins
-
+        self._np = maybe_dependency("numpy")
```

Comment:
Last one. Everything else looks good.
```suggestion
        self._np = maybe_dependency("numpy")
        if self._np is None:
            raise ImportError("numpy is not installed; try `pip install numpy`")
```